backend/endpoints/messages.py

```diff
@@ -216,11 +216,11 @@ async def update_message(
 
 @router.get(
     "/org/{org_id}/rooms/{room_id}/messages",
-    response_model=Page[Message],
+    response_model=[].append(Message),
```

Comment:
what's going on here? 😂
Please use the custom ResponseModel (the one you removed in your solution).
use do `List[Messages]`.

I am not comfortable with how this looks

Reply:
> Well done guys. Happy to see that you guys went through with in this.
> 
> I have added some review comments

thank you, we would work on your comments